frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerListItem.component.tsx

```diff
@@ -64,11 +69,16 @@ export const ContainerListItem = ({
 	if (container.hasStatsPending) {
 		return <SkeletonListItem delay={index / 10} key={container._id} />;
 	}
-
 	const { teamspace, project } = useParams<DashboardParams>();
+	const isMainList = useContext(IsMainList);
+
 	useEffect(() => {
-		enableRealtimeContainerUpdateSettings(teamspace, project, container._id);
-		enableRealtimeContainerRemoved(teamspace, project, container._id);
+		if (isMainList) {
+			enableRealtimeContainerUpdateSettings(teamspace, project, container._id);
+			enableRealtimeContainerRevisionUpdate(teamspace, project, container._id);
+			enableRealtimeNewRevisionUpdate(teamspace, project, container._id);
+			enableRealtimeContainerRemoved(teamspace, project, container._id);
```

Comment:
We should use the combine function that is in  for multiple subscriptions.  
(you can get the code from here, we'll fix the conflicts later)
https://github.com/3drepo/3drepo.io/blob/e63633b966e196fe270c7930025df8bf76add392/frontend/src/v5/services/realtime/realtime.service.ts#L127



What this function does is to combine all the unsubscriptions from the enable... functions and return one function that will be used as a cleanup for useeffect.

```suggestion
			return combineSubscriptions(
				enableRealtimeContainerUpdateSettings(teamspace, project, container._id),	
				enableRealtimeContainerRevisionUpdate(teamspace, project, container._id),
				enableRealtimeNewRevisionUpdate(teamspace, project, container._id),
				enableRealtimeContainerRemoved(teamspace, project, container._id),
			)
```

Reply:
Ahh, okay. Am I understanding this right that the unsubscribe function isn't being returned in the method I've done?

Reply:
yup